src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -726,9 +726,11 @@ public void setDataBean(DataBean dataBean) {
             url = new URL(this.dataBean.getWmsUrl());
         } catch (MalformedURLException e) {
         }
-        mapWFS = new WMSMapSwing(url, MAP_WIDTH, MAP_HEIGHT);
+        mapWFS = new WMSMapSwing(url, MAP_WIDTH, MAP_HEIGHT, this.dataBean
+                .getWMSLayer());
         mapWFS.setCoordinateDisplay(basicX1, basicY1, basicX2, basicY2);
-        mapAtom = new WMSMapSwing(url, MAP_WIDTH, MAP_HEIGHT);
+        mapAtom = new WMSMapSwing(url, MAP_WIDTH, MAP_HEIGHT, this.dataBean
+                .getWMSLayer());
```

Comment:
because the databean holds the information parsed from the serviceSettings.xml file